docs/.vuepress/components/bug-list.vue

```diff
@@ -16,7 +16,7 @@
                         ></span>
                     </button>
                     <p class="bug-header-title">
-                        <a :href="bug.html_url" target="_blank">{{
+                        <a :href="bug?.html_url" target="_blank">{{
```

Comment:
Why is the question mark necessary?

Reply:
The build did fail sometimes due to `bug` being undefined and `bug.user` being undefined in lines 49&50.
I couldn't find out when and why that happened but this prevents the build from failing when that happens.